HandlingResources.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+class HandlingResources
+{
+  private $xml_data = null;
+  public $init_arr = [];
+  public $data = [];
+
+  private function replaceKey($arr, $old_key, $new_key)
+  {
+    $keys = array_keys((array) $arr);
+    if (array_key_exists($old_key, (array) $arr)) {
+      $keys[array_search($old_key, $keys)] = $new_key;
+    }
+    return $arr = array_combine($keys, (array) $arr);
+  }
+
+  public function formatting($init_arr, $arr)

```

Comment:
for name convention, please use verbs for the function name and noun for the class name